docs/python-integration.md

```diff
@@ -1,3 +1,38 @@
+# Creating vectorizers from python
```

Comment:
If we are going to add this to the docs, I'd like at least some tests of using this functionality outside of Alembic